atlasdb-ete-tests/docker-compose.multiple-cassandra.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
```

Comment:
Kind of weird for these docker-compose files to live in the root directory. Consider moving them to docker/compose or similar.

(I'm happy for this to happen in a separate PR)